README.md

```diff
@@ -24,13 +24,15 @@ For more details, see [contribution guidelines](CONTRIBUTING.md).
 - [Lego](https://github.com/go-acme/lego) - A ACME client and library that supports Gandi V5 API and built in Go.
 - [LibDNS](https://github.com/libdns/libdns) - A Universal DNS provider APIs for Go with a [Gandi V5 provider](https://github.com/libdns/gandi) available.
 - [libcloud](https://github.com/apache/libcloud) - A Python library to manage cloud resources (compute and DNS for Gandi driver).
+- [Lexicon](https://github.com/AnalogJ/lexicon) - A Python library to manipulate DNS records on various DNS providers in a standardized way.
```

Comment:
I would write a gandi-oriented sentence for this tool, otherwise there are a lot of other tools supporting gandi that we could add, for example anything that integrates with openstack. Maybe something like:
`A Python library to manipulate DNS records in a standardized way, supporting Gandi LiveDNS`

Reply:
Applied your suggestion.